src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -28,8 +28,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ItemGroup>
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultNoneItems)' == 'true' ">
     <None Include="**/*" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    <None Remove="**/*$(DefaultLanguageSourceExtension)" Condition=" '$(EnableDefaultCompileItems)' == 'true' "/>
-    <None Remove="**/*.resx" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' "/>
+    <None Remove="**/*$(DefaultLanguageSourceExtension)" />
+    <None Remove="**/*.resx" />
```

Comment:
Q: Does this traverse the project tree three times. I realize we're already doing it in common case. cc @cdmihai